handlers/hjt/hjt.go

```diff
@@ -0,0 +1,71 @@
+package hjt
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/lordralex/absol/database"
+	"github.com/lordralex/absol/logger"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+func RunCommand(ds *discordgo.Session, mc *discordgo.MessageCreate, c *discordgo.Channel, cmd string, args []string) {
+	if len(args) == 0 {
+		return
+	}
+
+	pasteLink := args[0]
+
+	content, err := readFromUrlAsLowercase(pasteLink)
+	if err != nil {
+		_, err = ds.ChannelMessageSend(c.ID, "Invalid URL")
+		logger.Debug().Printf("Invalid url: %s\n%s", pasteLink, err)
+		return
+	}
+
+	db, err := database.Get()
+	if err != nil {
+		_, err = ds.ChannelMessageSend(c.ID, "Failed to connect to database")
+		logger.Err().Printf("Failed to connect to database\n%s", err)
+		return
+	}
+
+	var data []hjt
+	var values []string
+	err = db.Find(&data).Where("? LIKE CONCAT('%', LOWER(name) ,'%')", content).Pluck("value", &values).Error
```

Comment:
This now has a pointless struct, you can use .Table to specify the table and not need your struct definition (also means you control the table name)

Reply:
done